foreshadow/intents/intent_type.py

```diff
@@ -7,9 +7,9 @@ class IntentType:
     NUMERIC = "Numeric"
     CATEGORICAL = "Categorical"
     TEXT = "Text"
-    OTHER = "Other"
+    Droppable = "Droppable"
```

Comment:
Shall we follow the same convention to keep all letters capital?

Reply:
Will do.